.gitignore

```diff
@@ -17,6 +17,7 @@
 *.blg
 *.lot
 *.exo
+*.mod
```

Comment:
`.mod` files are similar to pre-compiler C headers. They are generated by Fortran. Right now, the are generated in the source tree. I think that through some CMake magic it could be done in build, but I have not investigate that.